index.js

```diff
@@ -152,30 +192,15 @@ function sendNotification(endpoint, params) {
 
       if (vapid && !isGCM) {
```

Comment:
One thing I didn't make clear as well - when you subscribe passing in the applicationSeverKey in Chrome, you get an FCM endpoint rather than a GCM endpoint, this is what causes isGCM to be false resulting in this code path, allowing FCM to authenticate the applicationServerKey